heron/spi/src/java/com/twitter/heron/spi/packing/PackingPlanProtoDeserializer.java

```diff
@@ -0,0 +1,60 @@
+// Copyright 2016 Twitter. All rights reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//    http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+package com.twitter.heron.spi.packing;
+
+import java.util.HashMap;
+import java.util.Map;
+
+import com.twitter.heron.proto.system.PackingPlans;
+
+/**
```

Comment:
Instead of two classes - one for serialization and the other deserialization - can we have all of them in a single class? It will be easy to read and any modification will be localized - makes it easy to read and review code - in case one is changed but not the other?